Homework 8 - Chapter 9 - Nguyen Chinh Quan - 220066

In [3]:
import pandas as pd
import math

9.76.

In [18]:
mean_76 = 35
n_76 = 41
sample_mean_76 = 36.22
variance_76 = 5.7

a. alpha = 0.01 -> z(alpha) = 2.33


H0: mean <= 35, Ha: mean > 35

In [20]:
zobs_76 = (sample_mean_76-mean_76) / (math.sqrt(variance_76)/math.sqrt(n_76)) 
zobs_76

3.272007806793755

We see that 3.27 >= 2.33, therefore, there is evidence to suggest the mean LOA has increased

b. z(0.1) < z(0.01) (1.28 < 2.33), therefore, the answer will not change.

9.143.

In [4]:
data_143 = pd.read_excel(r"D:\stat\Tex-_Datasets\Chapter 9\ex09-143flights.xls")
data_143.describe()

Time (min)
count   15.000000
mean   310.933333
std     13.677232
min    297.000000
25%    300.500000
50%    307.000000
75%    316.000000
max    348.000000

In [5]:
data_143.value_counts().sum() #df = 14

15

H0: mean <= 303, Ha: mean > 303

alpha = 0.025

In [6]:
t_alpha_143 = 2.145
t_obs_143 = (310.93 - 303) / (13.68 / math.sqrt(15)) 
t_obs_143

2.245084644402401

t_obs >= t_alpha (2.245 >= 1.96). Therefore, there is evidence to suggest the true mean air time is greater than this 303 minutes.

9.156.

H0: mean >= 15, Ha: mean < 15

a.

In [7]:
n_156 = 11 #df = 10
mean_156 = 14.35
std_156 = 3.75

alpha = 0.025

In [8]:
t_alpha_156 = 2.228
t_obs_156 = (mean_156 - 15) / (std_156 / math.sqrt(n_156))
t_obs_156

-0.5748816303282696

We see that -0.58 > -2.228. Therefore, there is no evidence to refute the manager's claim.

b. We do not reject the null hypothesis when $14.35 is certainly less than $15.00 is because it maybe due the sample's variability and small sample size.

c. Using the t table for df = 10 and one-tail test, we can find out the bounds on the p value is 0.25 and 0.5

9.181.

a.

In [9]:
n_181 = 225
x_181 = 189
p0_181 = 0.9
p_181 = x_181/n_181

b.

In [10]:
np0_181 = n_181*p0_181
n_1_minus_p0 = n_181*(1-p0_181)
print(np0_181, n_1_minus_p0)

202.5 22.499999999999996


Both values are >= 5, therefore the large-sample test is appropriate.

c. alpha = 0.05 -> z(alpha/2) = z(0.025) = -1.96 

H0: p = 0.9, Ha: p != 0.9

In [12]:
zobs_181 = (p_181 - p0_181) / math.sqrt(p0_181*(1-p0_181)/n_181)
zobs_181

-3.000000000000003

We have -3 <= -1.96, therefore we have evidence to reject the null hypothesis.

d. Using online z-calculator to calculate the p value from z value, we have the value of p = 0.0027

9.223.

In [14]:
variance_223 = 22.5
n_223 = 37
sample_variance_223 = 15.6

H0: variance >= 22.5, Ha: variance < 22.5

alpha = 0.05

In [17]:
x_squared_223 = (n_223-1)*sample_variance_223/variance_223
x_squared_223 

24.96

Using table VI from the appendix, we have: x^2(1-0.05, 36) = 23.2686 

We see that 24.96 > 23.2686, therefore there is no evidence to suggest that the bull riding has become less exciting.